lib/cli/commands/listpeers.ts

```diff
@@ -52,12 +48,9 @@ const formatPeers = (peers: ListPeersResponse.AsObject) => {
     formattedPeers.push([
       peer.address,
       trimPubKey(peer.nodePubKey),
-      `${peer.inbound}`,
-      peer.xudVersion,
-      `${peer.secondsConnected.toString()} seconds`,
       formatPairList(peer.pairsList),
-      formatLndPubKeys(peer.lndPubKeysMap),
-      trimPubKey(peer.raidenAddress),
+      // tslint:disable-next-line:max-line-length
+      `inbound: ${peer.inbound}\nversion: ${peer.xudVersion}\ntime connected: ${peer.secondsConnected.toString()} seconds\nlnd keys: ${formatLndPubKeys(peer.lndPubKeysMap)}\nraiden address: ${trimPubKey(peer.raidenAddress)}`,
```

Comment:
Two more thoughts.

1. Can we put each lnd key on a separate line? Like

```
BTC lnd pubkey: 02d693...
LTC lnd pubkey: 02572a...
```

2. The `raiden address` line has some space, and of the 6 characters we show before cutting off, the first two are `0x` leaving only 4 unique characters. I think it would be better to show 8 (or maybe even 10) total characters for the raiden address, including the `0x`.